browser/resources/settings/brave_content_page/background_colors.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { hexColorToSkColor } from 'chrome://resources/js/color_utils.js'
+
+// TODO(sko): Is it possible to access leo variable from js side here?
```

Comment:
unfortunately, not easily - you could try including `node_modules/@brave/leo/tokens/index.ts` in the `mojo_files` of your BUILD.gn, but I don't know where it'd end up :D 

The other alternative would be to try reading them from `body.style`

```ts
const color = document.getComputedStyle(body).getProperty('--leo-color-XXX')
```